pkg/minishift/openshift/service.go

```diff
@@ -72,7 +89,7 @@ func GetServiceURL(service, namespace string, https bool) (string, error) {
 }
 
 // Get the available routes to user
-func GetServiceURLs(serviceListNamespace string) ([]ServiceURL, error) {
+func GetServiceURLs(serviceListNamespace string, ip string) ([]ServiceURL, error) {
```

Comment:
I think we should rename the _ServiceURL_ struct. It contains much more than a URL to a service. Maybe _ServiceSpec_?